LuckySpin/Services/SpinService.cs

```diff
@@ -15,11 +15,18 @@ public SpinService(ISpinRepository sr)
 
         }
 
-
-        public double CalculateAvgWins()
+        public double CalculateAvgWins(bool winning)
         {
-            //TODO: Write logic to use the "real" spinRepository NOT the test data
-            return .1; 
+            //TODO: Write logic to use the "real" spinRepository NOT the test data'
+            double wins = 0;
+            if (winning)
+                wins = 1;
```

Comment:
Consider using the conditional operator `?:` to assign the wins start value which takes a boolean and returns either the first if true or second value if not. 
`double wins = winning ? 1 : 0;`